cdap-docs/tools/docs-cli-commands.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#
+# Copyright © 2016 Cask Data, Inc.
+#
+# Used to generate a table in the CLI documentation from the output of CLI tools.
+# 
+# If input_file is not found, puts out a warning message in the log and puts a note 
+# in the table that is generated.
+#
+
+from optparse import OptionParser
+import os
+import sys
+
+VERSION = "0.0.1"
+
+LITERAL = '``'
+SPACES = ' ' * 3
```

Comment:
It's 3 because that's what the CLI doc tool generates, and I'd need to change it there...